nservicebus/vscode.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Debugging NServiceBus in Visual Studio Code
+summary: How to configure Visual Studio Code to build and debug multiple NServiceBus endoints simultaneously
+component: Core
+reviewed: 2018-08-01
+---
+
+This article describes how to configure VSCode to build an NServiceBus solution with multiple projects and debug multiple endpoints simulateously.
```

Comment:
This "VSCode" should be where you use the full name and then shorten it, instead of line 10.

Reply:
simulateously -> simultaneously